platform-includes/getting-started-install/react-native.mdx

```diff
@@ -33,8 +27,6 @@ If you're using Expo, [read our docs](/platforms/react-native/manual-setup/expo/
 
 ### iOS Specifics
 
-The SDK needs access to certain information about the device and the application for its essential functionality. Some of the APIs required for this are considered privacy-relevant. Add the privacy manifest to your Xcode project to ensure your app is compliant with Apple's guidelines. Read the [Apple Privacy Manifest](/platforms/react-native/data-management/apple-privacy-manifest/) guide for more info on how to add records required for the Sentry SDKs.
-
 When you use Xcode, you can hook directly into the build process to upload debug symbols and source maps.
 
 ### Android Specifics
```

Comment:
Both of these sections can be removed. 

The information about the build phases is included in the wizard description.

The Android info is also outdated `react-native link` command doesn't exist any more, the info is not relevant anymnore.

The `configureondemand` on demand can be moved to troubleshooting.